swagger/forward/swagger.yaml

```diff
@@ -0,0 +1,92 @@
+swagger: '2.0'
+info:
+  title: Mailgun Relayery Forward Server API
+  description: |
+    API for the server listening to the remote instances' push of messages and forwarding them to MailGun.
+
+    The instances need to provide an authentication token upon pushing to this server.
+  version: "1.0.0"
+host: mailgun.relayery.forward.parquery.com
+schemes:
+  - https
+basePath: /
+tags:
+- name: "forward"
+paths:
+  /api/message:
+    put:
+      operationId: put_message
+      tags:
+        - forward
+      description: |
+        sends a message to the server, which forwards it to the MailGun API.
+
+        The given (descriptor, token) pair are authenticated first.
```

Comment:
You can remove the descriptor and token from the parameters since they are provided as basic auth. In the implementation you need to extract them from the header.